native/cocos/math/Color.cpp

```diff
@@ -93,6 +97,13 @@ void Color::set(const Color &p1, const Color &p2) {
     a = p2.a - p1.a;
 }
 
+void Color::set(const Vec4 &v) {
+    r = v.x * 255.F;
+    g = v.y * 255.F;
+    b = v.z * 255.F;
+    a = v.w * 255.F;
+}
```

Comment:
Revert the changes in Color.h/.cpp, they are not needed now.

Reply:
ok, this change reverted.